src/mesh/duplicate_nodes.h

```diff
@@ -0,0 +1,151 @@
+/*!
+ *
+﻿ * Copyright (C) 2015 Technical University of Liberec.  All rights reserved.
+ * 
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License version 3 as published by the
+ * Free Software Foundation. (http://www.gnu.org/licenses/gpl-3.0.en.html)
+ * 
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+ *
+ * 
+ * @file    duplicate_nodes.h
+ * @brief   
+ */
+
+#ifndef DUPLICATE_NODES_H
+#define DUPLICATE_NODES_H
+
+
+
+
+class Mesh;
+
+template<int dim>
+class MeshObject {
+public:
```

Comment:
Document this class, probably it represents any edge, face or element (n-face) of the mesh in terms of 
(n-1)-face.